poetry/core/masonry/builders/builder.py

```diff
@@ -36,7 +41,7 @@ class Builder(object):
 
     def __init__(
         self, poetry, ignore_packages_formats=False
-    ):  # type: ("Poetry", bool) -> None
+    ):  # type: (Poetry, bool) -> None
```

Comment:
I think we should keep the quotes here and in the other places. `Poetry` isn't imported during runtime. While this works in the "python2 way" of type hinting, the quotes are needed in the "python3 way". We might want to switch to this style, once python2 support is dropped. Keeping the quotes makes it easy to use automatic converter for this step.

Reply:
Good catch.